pkg/apiserver/certificate/cacert_controller.go

```diff
@@ -59,6 +63,16 @@ type CACertController struct {
 
 var _ dynamiccertificates.Listener = &CACertController{}
 
+func GetCAConfigMapNamespace() string {
+	namespace := env.GetPodNamespace()
+	if namespace != "" {
+		return namespace
+	}
+
+	klog.Warning("Failed to get Pod Namespace from environment. Use \"kube-system\" as the CA ConfigMap Namespace")
+	return "kube-system"
```

Comment:
do we tend to use constants for these defaults in the rest of the code base?

Reply:
Defined a constant.